src/event/event.controller.ts

```diff
@@ -133,6 +153,8 @@ export class EventController {
     );
   }
 
+  @Permissions(EventAttendeePermission.ManageAttendees)
```

Comment:
It depends on event/group visibility if members/attendees are available. Limiting this method with guard will prevent fetching members on public events.